js/render.js

```diff
@@ -0,0 +1,184 @@
+//pick random colors from the array of frog colors
+const generate_random_frog_color = function()
```

Comment:
The names of your functions and variables are a bit long. I would try to shorten them. 
`getFrogColor` might be better and easier to read through.